server/scripts/stats.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable */
+const { dbClient } = require('../db/db.js');
+const { collections } = require('../db/schema.js');
+const inquirer = require('inquirer');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+require('dotenv').config({ path: '../.env' });
+
+dayjs.extend(utc);
+
+const toPostgresDate = (date) => {
+  return dayjs(date).utc().format('YYYY-MM-DD HH:mm:ssZ');
+}
+
+/* eslint-disable no-console */
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+
+      const healthAuthorities = [
```

Comment:
Just a note since I ran the report on prod data last night, we already have some submissions that came in without this field being populated (since we relaxed the validation to allow that to happen). So we might want to add a blank string here as well, just to make it easy to report on those numbers rather than needing to math it out :)

Reply:
True. Nice catch.